syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -122,6 +124,14 @@ def get(self):
 
         return result
 
+    def mid_get(self):
```

Comment:
Not exactly sure, because it require .get() to be implemented in abstract.py which is not the case.
But maybe .get() should also have a default implementation in abstract.py I agree.
I'll look at it and try to do this!